### DISCUSSION TITLE
Reactivity transform

### DISCUSSION THREAD

**Discussion by kylegl** [vueuse/vueuse#2213] [Q&A]
Hello, 

I see that they introduced a new convention in v9 to work better with reactivity transform. I don't see any examples using the syntax $( ). 

I'm guessing its something like this:

```
const isTrue = $ref(false)

const toggle = useToggle($$(isTrue))
```
and then say I wanted the result to utilize reactivity transform. For example:

```
const original = 'some value'
const { cloned } = $(useClone(original))
```
Any insight would be helpful.

Thanks!

**Comment by antfu** [comment]
Yeah I initially thought it had been covered by Vue's docs. But indeed it would be great if we mention it a bit in our docs as well. PR welcome!

**Reply by kylegl** [reply]
Ok, thanks for the response. I'll play around with it and make a PR if all goes well.
